README.md

```diff
@@ -1054,52 +1349,77 @@ migrate <targetDb> <service> <subservice>
 iotaCommands.init(configCb, configIot);
 ```
-* Add the IOTA Lib commands to your array of commands
-```
+
+-   Add the IOTA Lib commands to your array of commands
+
+```javascript
 commands = commands.concat(commands, iotaCommands.commands);
 ```
-* Execute the command line interpreter as usual:
-```
+
+-   Execute the command line interpreter as usual:
+
+```javascript
 clUtils.initialize(commandLine.commands, 'IoT Agent tester> ');
 ```
 
-The command line module makes use of two configuration objects. Both can be shown and edited in the command line using the
-provided commands, but a default value must be present.
+The command line module makes use of two configuration objects. Both can be
+shown and edited in the command line using the provided commands, but a default
+value must be present.
 
-The Context Broker configuration object holds all the information about the Context Broker where the IoT Agent to be tested
-is connected. It MUST contain the following attributes:
-* **host**: host where the Context Broker instance is located.
-* **port**: port where the Context Broker instance is listening.
-* **service**: service that will be used in all the NGSI operations.
-* **subservice**: service that will be used in all the NGSI operations.
+The Context Broker configuration object holds all the information about the
+Context Broker where the IoT Agent to be tested is connected. It MUST contain
+the following attributes:
 
-The IoT Agent configuration object holds information about the IoT Agent that is being tested. It MUST contain the following
-attributes:
-* **host**: host where the IoT Agent instance is located.
-* **port**: port where the IoT Agent instance is listening.
-* **service**: service that will be used to group devices and device information.
-* **subservice**: subservice that will be used to group devices and device information.
+-   **host**: host where the Context Broker instance is located.
+-   **port**: port where the Context Broker instance is listening.
+-   **service**: service that will be used in all the NGSI operations.
+-   **subservice**: service that will be used in all the NGSI operations.
+
+The IoT Agent configuration object holds information about the IoT Agent that is
+being tested. It MUST contain the following attributes:
+
+-   **host**: host where the IoT Agent instance is located.
+-   **port**: port where the IoT Agent instance is listening.
+-   **service**: service that will be used to group devices and device
+    information.
+-   **subservice**: subservice that will be used to group devices and device
+    information.
+
+---
 
-##  <a name="development"/> Development documentation
+## Licence
 
-Information about developing for the IoTAgent Library can be found at the corresponding section of the [User & Programmers Manual](doc/usermanual.md).
+The IoT Agent Node Library is licensed under Affero General Public License (GPL)
+version 3.
 
+© 2018 Telefonica Investigación y Desarrollo, S.A.U
```

Comment:
Alternatives for this:

1. © Telefónica Investigación y Desarrollo, S.A.U (the one currently in the PR)
2. © 2014-2018 Telefónica Investigación y Desarrollo, S.A.U (given the project was created in 2014)
3. © Telefónica Investigación y Desarrollo, S.A.U (to avoid being obsolete each time a new year comes)

I'm not a lawyer :) so I don't know if there are different implications depending the case, but my prefereed one is alternative 3.

Reply:
NTC -  This is U.S. standard for implementation of the Berne convention

> © year name

The single year which is updated with each release also indicates when the individual **release** will fall out of copyright. Year ranges have no legal meaning.

 see - https://opensource.stackexchange.com/a/2875

Reply:
Very interesting post :) Thanks for the link!

However, two things catch my attention:

* _"The Berne Convention leaves "implementation details" up to individual countries"_. The rest of the post refers to US legislation. Should legistlation in Europe (and particularly Spain, which I understand is the law that applies to this case) the same or the "implementation details" are different?
*  _"In practice new software releases are considered similarly to derivatives, so the copyright notices accumulate; and when the name of the owner doesn't change, notices are merged, e.g. "© 2014, 2016""_. I understand this opens a 4th variant as follows:

4. © 2014, 2015, 2016, 2017, 2018 Telefónica Investigación y Desarrollo, S.A.U

Let me cast an internal query to our internal legal department to see what the think about this.

(A bit off-topic, but note that FIWARE Lab portal is using the "range year form", https://lab.fiware.org. Should they fix?)

Reply:
> Let me cast an internal query to our internal legal department to see what the think about this.

Our legal departement also recommends option 1, so that will be the one we will use at the end.

It will involve some extra work to change the number at new year, but we already are doing that kind of thing in other repos (e.g. https://github.com/telefonicaid/fiware-orion/pull/3067) so it wouldn't be a big problem.

Reply:
NTC